src/components/AuthContainer.js

```diff
@@ -38,6 +39,50 @@ export default class AuthContainer extends Component {
       }, () => {
         console.warn('Error looking up user status')
       })
+
+      Rawls.listBillingProjects().then(
+        billingProjects => {
+          this.billingProjects = billingProjects
+          this.checkClusters()
+        },
+        failure => Utils.log('Error fetching billing projects', failure)
+      )
+
+      Leo.clustersList().then(
+        clusters => {
+          this.clusters = clusters
+          this.checkClusters()
+        },
+        failure => Utils.log('Error fetching clusters list', failure)
+      )
+    }
+  }
+
+  checkClusters = () => {
+    const { billingProjects, clusters } = this
+    if (billingProjects && clusters) {
```

Comment:
In a case like this, where you have several independent async operations, you can use `Promise.all` to launch them all in parallel, and the master promise will resolve when they've all completed. So in this case you could do:
```
const [billingProjects, clusters] = await Promise.all([Rawls.listBillingProjects(), Leo.clustersList()])
// continue processing
```